dev/AppNotifications/AppNotificationTelemetry.h

```diff
@@ -20,16 +21,16 @@ class AppNotificationTelemetry : public wil::TraceLoggingProvider
         winrt::hresult hr,
         std::wstring const& appId) noexcept try
     {
-        if (c_maxEventLimit >= UpdateLogEventCount())
+        if (m_telemetryHelper.ShouldLogEvent())
```

Comment:
cool. Looks good to me. Whats the advantage of moving to a helper? Reusability for other Telemetry providers?

Reply:
We already had  this code in two separate providers and the upcoming AppNotificationBuilderTelemetryProvider will also need access to this code.